CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.6.0
```

Comment:
```suggestion
## 0.5.1
```

This has no breaking changes whatsoever. It could even be a patch (`5.0.0+1`), but I think it is fair to use a minor version bump due to the deprecations.

Reply:
See https://github.com/axel-op/feature_discovery/pull/24.